ci_group/revolve2/ci_group/morphological_measures.py

```diff
@@ -93,10 +99,10 @@ def __init__(self, body: Body) -> None:
 
         :param body: The body to measure.
         """
-        self.body_as_grid, self.core_grid_position = body.to_grid()
-
-        self.is_2d = self.__calculate_is_2d(body)
+        grid, self.core_grid_position = body.to_grid()
+        self.grid = np.asarray(grid)
```

Comment:
once the grid cleanup is in a good stat i would like to make it return a np array. This makes operations on it faster, and easier to debug.